db/data_migration/20200218104101_remove_brexit_cta_tag_from_withdrawn_docs.rb

```diff
@@ -0,0 +1,11 @@
+withdrawn_brexit_cta_editions = Edition.in_default_locale
```

Comment:
Will this allow us to 💥 `BrexitCTA` that are in other languages, like welsh?
Are there any?

Reply:
So after some digging, turns out `Edition.joins(:translations).includes(:document).where("edition_translations.body LIKE ?", "%$BrexitCTA%")` and `Edition.in_default_locale.includes(:document).where("edition_translations.body LIKE ?", "%$BrexitCTA%")` return the same list.

In other words, all `$BrexitCTA` tags are in the English translations, which happens to be the default locale. So we're good.

Reply:
ace, thanks for chasing that!